mssql-plugin/src/e2e-test/features/mssql/mssql sink/DesignTime.feature

```diff
@@ -0,0 +1,76 @@
+# Copyright © 2023 Cask Data, Inc.
+##
+## Licensed under the Apache License, Version 2.0 (the "License"); you may not
+## use this file except in compliance with the License. You may obtain a copy of
+## the License at
+##
+## http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+## WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+## License for the specific language governing permissions and limitations under
+# the License..
+
+@Mssql
+Feature: MySQL Server Sink - Design time scenarios
+
+  @MSSQL_SOURCE_TEST @MSSQL_SINK_TEST @Mssql_Required
```

Comment:
We don't need @Mssql_Required tag in each scenario

Reply:
resolved